### PR TITLE
I159 weightings in json file

### PR DESCRIPTION
**Pull request by mataszilaitis**
We can now read weightings from .json files, which are located in `FeatureWeightings` folder

### CODE REVIEW

File: YoCode/FeatureDetailsStorage.cs [hunk 1]
```diff
@@ -1,172 +1,216 @@
...
 namespace YoCode
 {
     class FeatureDetailsStorage
     {
-        public Dictionary<Feature,FeatureDetails> InitializeJuniorDetails()
+        Dictionary<Feature, double> WeightingsFromJson;
+        TestType mode;
+
+        string JuniorPath = @"..\..\..\FeatureWeightings\JuniorWeightings.json";
+        string OriginalPath = @"..\..\..\FeatureWeightings\OriginalWeightings.json";
```
Comment (jogobom) [inline YoCode/FeatureDetailsStorage.cs]: One downside of this is that we now have extra files that we need to ship with YoCode, right? We must remember to include them in the zip file. In fact, do these paths need to be in `appsettings.json`? I can imagine they will be different when we package them in the zip file.

Comment (mataszilaitis) [reply YoCode/FeatureDetailsStorage.cs]: These are stored in FeatureWeightings folder.
image

It contains two .json files, JuniorWeightings and OriginalWeightings. Why is that a problem?

Comment (jogobom) [reply YoCode/FeatureDetailsStorage.cs]: Where will they go in the zip file when we release YoCode? Will `..\..\..\FeatureWeightings` be the right directory in that case?

Comment (mataszilaitis) [reply YoCode/FeatureDetailsStorage.cs]: Probably not... What's the possible solution?

Comment (jogobom) [reply YoCode/FeatureDetailsStorage.cs]: `appsettings.json`. Get the name of the directory from `appsettings.json`, then the zip file can contain an `appsettings.json` that we've already set up with the right paths. The weightings files can be in git just as you've done, that's fine, but the path that YoCode uses to find them should come from `appsettings.json`.

Comment (jogobom) [reply YoCode/FeatureDetailsStorage.cs]: It was easier when nobody had suggested being able to edit the weightings. ;)

Comment (mataszilaitis) [reply YoCode/FeatureDetailsStorage.cs]: Nothing that is worth doing is easy 😁

**Pull request merged**